packages/interface-stream-muxer-compliance-tests/src/close-test.ts

```diff
@@ -67,6 +67,48 @@ export default (common: TestSetup<StreamMuxerFactory>) => {
       expect(dialer.streams).to.have.lengthOf(0)
     })
 
+    it('calling close closes streams', async () => {
```

Comment:
Needs a test where an error is passed to `.close` to verify the correct behaviour.